### PR TITLE
Dlt 3 create flyway scripts

### PR DESCRIPTION
**Pull request by DavidBudnitsky-BBD**
# Added flyway scripts

## Type of change

- [ ] New feature
    - DB is in dev, should be in prod after the PR is approved.

### CODE REVIEW

File: .github/workflows/db-prod-ci-cd.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+name: "Database CICD Prod"
```
Comment (ryanbasiltrickett) [inline .github/workflows/db-prod-ci-cd.yaml]: This needs to be combined with the proj-ci-cd else the pipeline will not be sequential

File: .github/workflows/db-dev-ci-cd.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+name: "Database CICD Dev"
+ 
+on:
+  push:
+    branches:
+        - '*'
+        - '!main'
```
Comment (ryanbasiltrickett) [inline .github/workflows/db-dev-ci-cd.yaml]: This needs to be combined with the proj-ci cause we dont want deployment every time you upload that means that two people can cause changes to the db at once

**Review by ryanbasiltrickett — CHANGES_REQUESTED** [review]

File: .github/workflows/db-prod-ci-cd.yaml (hunk 1 above)
Comment (DavidBudnitsky-BBD) [reply .github/workflows/db-prod-ci-cd.yaml]: I agree with this

File: .github/workflows/db-dev-ci-cd.yaml (hunk 2 above)
Comment (DavidBudnitsky-BBD) [reply .github/workflows/db-dev-ci-cd.yaml]: I agree

File: SQL/V202404051657__CreateTables.sql [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  Language NVARCHAR(50),
+  Abbreviation VARCHAR(5)
+);
+
+CREATE TABLE Documents (
+  DocumentID INT IDENTITY(1,1) PRIMARY KEY,
+  UserID INT,
+  LanguageID INT,
+  GenTime DATETIME DEFAULT CURRENT_TIMESTAMP,
+  Path NVARCHAR(255),
```
Comment (ryanbasiltrickett) [inline SQL/V202404051657__CreateTables.sql]: no path needed, more the need for a doc name

**Review by ryanbasiltrickett — CHANGES_REQUESTED** [review]

**Review by ryanbasiltrickett — APPROVED** [review]
LFTM

**Pull request merged**